packages/common/src/core/types/IHubSite.ts

```diff
@@ -79,6 +79,11 @@ export interface IHubSite
 
   /** Legacy teams - list of ids */
   legacyTeams: string[];
+
+  /**
+   * Whether or not the site is an org site--should be removed when no longer necessary
```

Comment:
I would maybe slightly amend this comment to read something like: `True when the site is a "Hub Home" site`. We probably wouldn't want to remove this, as it'd technically be a breaking change and other features might want/need to cue off of this in the future.